### ISSUE TITLE
Please add LICENSE file

### ISSUE THREAD

**Issue by gmacario** [gocd/documentation#203]
According to the [GoCD CLA](https://www.go.cd/contribute/cla.html)

> ThoughtWorks is promising to you that if it distributes your contributions, ThoughtWorks will distribute them under an OSI-approved open source license.

Please make it explicit under which license you will be distributing the documentation by adding a `LICENSE` file in the top of this repository.

**Comment by ketan** [comment]
Thank you for bringing this to our notice, I or someone on the core team will get back to you on this in a few days. We are treating this issue on high priority.

**Comment by ketan** [comment]
Hi @gmacario -

Thank you for reporting this issue, and I'm sorry it took longer than expected to get back to you on this.

After having spoken to the legal team, we've decided to go ahead with adopting the Apache-2.0 license for the documentation website (inline with the rest of the source code). I've just merged a PR to include a [`LICENSE`](https://github.com/gocd/documentation/blob/master/LICENSE) file at the project root and including it in the [`README`](https://github.com/gocd/documentation/blob/master/README.md#license) file as well

I'm going to mark this issue as closed.
